DemoApp/DemoAppCoordinator.swift

```diff
@@ -441,3 +470,16 @@ class CustomChatMessageActionsVC: ChatMessageActionsVC {
         }
     }
 }
+
+extension MessageReactionType {
+    var position: Int {
+        switch rawValue {
+        case "love": return 0
+        case "haha": return 1
+        case "like": return 2
+        case "sad": return 3
+        case "wow": return 4
+        default: return 5
+        }
+    }
+}
```

Comment:
Having `MessageReactionType: Comparable` can make it clear what the `position` means and would allow hiding the implementation details of sorting.

```suggestion
extension MessageReactionType: Comparable {
    public static func < (lhs: MessageReactionType, rhs: MessageReactionType) -> Bool {
        lhs.position < rhs.position
    }
    
    private var position: Int {
        switch rawValue {
        case "love": return 0
        case "haha": return 1
        case "like": return 2
        case "sad": return 3
        case "wow": return 4
        default: return 5
        }
    }
}
```

Reply:
Makes sense 👍

Reply:
@evsaev This is actually not possible since we need to pass the closure sorting to the `sorted()` function. This would work if we just called `sorted()` without any argument. But we can't do this. We would need to provide a default `Comparable` in our SDK, and then this wouldn't be overridable in the customer app. So the code will remain as is.